src/api.ts

```diff
@@ -125,54 +104,53 @@ function getExploreResult(
   results: (ExploreBundleResult | ExploreErrorResult)[],
   options: ExploreOptions
 ): ExploreResult {
-  const bundles: ExploreBundleResult[] = [];
-  const errors: ExploreErrorResult[] = [];
-
-  results.forEach(result => {
-    if ((result as ExploreBundleResult).files) {
-      bundles.push(result as ExploreBundleResult);
-    } else {
-      errors.push(result as ExploreErrorResult);
-    }
-  });
+  const [bundles, errors] = partition(
+    results,
+    result => !!(result as ExploreBundleResult).files
```

Comment:
`result => 'files' in result` and drop the cast

Reply:
Done